src/Core/Managed/Shared/DataContracts/RequestTelemetry.cs

```diff
@@ -222,7 +217,17 @@ void ITelemetry.Sanitize()
             this.Data.id = this.Data.id.SanitizeName();
             this.Data.id = Utils.PopulateRequiredStringValue(this.Data.id, "id", typeof(RequestTelemetry).FullName);
 
-            this.ResponseCode = Utils.PopulateRequiredStringValue(this.ResponseCode, "responseCode", typeof(RequestTelemetry).FullName);
+            // Required field
+            if (string.IsNullOrEmpty(this.ResponseCode))
```

Comment:
It is not initialized by default. If it is not autogenerated item you can use parameterless constructor and forget to initialize StatusCode. But as it is a required field we initialize it is sanitize. (Before we were setting it in parameterless constructor and now we move initialization in Sanitize method that is called before Send. This way in Web SDK we may check if request status code was initialized from customer's code and do not override it)